pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -40,7 +40,11 @@ const (
 
 	VirtAPIName        = "virt-api"
 	VirtControllerName = "virt-controller"
-	VirtOperatorName   = "virt-operator"
+	VirtOperatorName   = "virt-operator"	
+	PrometheusUrl	   = "prometheus.kubevirt.io"
+	KubeVirtUrl	   = "kubevirt.io"
+	KubernetesUrl	   = "kubernetes.io/hostname"
```

Comment:
This is a common label which can be used in affinity rules to select specific nodes. Consider naming it `KubernetesHostnameLabel`.